plugins/com.google.cloud.tools.eclipse.appengine.newproject/helpContexts.xml

```diff
@@ -1,24 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE contexts [
-  <!ENTITY service "&lt;b&gt;App Engine Service:&lt;/b&gt; An App Engine application is made up of one or more services. By using multiple services, you can deploy your app as a set of microservices with different performance settings. Each service must have a unique name. Do not reuse names  between services in the same application.">
+  <!ENTITY service "&lt;b&gt;App Engine Service:&lt;/b&gt; An App Engine application is made up of one or more services. By using multiple services, you can deploy your app as a set of microservices with different performance settings. Each service must have a unique name. Do not reuse names between services in the same application.">
 
   <!ENTITY project "&lt;b&gt;Project name:&lt;/b&gt; An Eclipse project name.">
 
+  <!ENTITY mavencheckbox "&lt;b&gt;Create as Maven project:&lt;/b&gt; Add a pom.xml file to the project and use the Maven builder.">
+
   <!ENTITY group-id "&lt;b&gt;Group ID:&lt;/b&gt; The Maven Group ID. Should be an alphanumeric path separated by periods. Typically the same as the Java package name.">
 
   <!ENTITY artifact-id "&lt;b&gt;Artifact ID:&lt;/b&gt; The Maven Artifact ID. Should be an alphanumeric name separated by dashes.">
+
+  <!ENTITY version "&lt;b&gt;Version:&lt;/b&gt; The Maven version. Best in the form &lt;major version>.&lt;minor version>.&lt;incremental version>-&lt;qualifier>">
```

Comment:
Worth prefixing as `maven-version` (and `group-id` and `artifact-id` too)?  I initially confused the `&version` in the flex definition as the Java-7/8 selector.